CLI_sort_conflict.py

```diff
@@ -0,0 +1,74 @@
+'''
```

Comment:
Oh I thought you were trying to make a new convention since the suggested name back then uses capital letters. I'll change it back to small letters next time.